pkg/util/duration/duration.go

```diff
@@ -30,6 +30,8 @@ const (
 	daysInMonth   = 30
 	nanosInDay    = 24 * int64(time.Hour) // Try as I might, couldn't do this without the cast.
 	nanosInMonth  = daysInMonth * nanosInDay
+	nanosInHour   = nanosInMinute * 60
```

Comment:
These should be moved to somewhere in duration_test.go since they are only used in tests.

Reply:
Thank you. I've fixed this problem.